command/server.go

```diff
@@ -566,6 +561,10 @@ func (c *ServerCommand) runRecoveryMode() int {
 		c.UI.Error(fmt.Sprintf("Error setting up seal: %v", setSealResponse.sealConfigError))
 		return 1
 	}
+	if setSealResponse.unwrapSeal != nil {
+		c.UI.Error("Recovry mode cannot be started with configuration for seal migration")
```

Comment:
```suggestion
		c.UI.Error("Recovery mode cannot be started with configuration for seal migration")
```

Reply:
Ugh, three lines of code and I manage to misspell something! :-(